### ISSUE TITLE
show average pick time for a card

### ISSUE THREAD

**Issue by TaZi-loh** [Razviar/mtgap#109]
**Detailed context:**
Description of your idea:
Add a specific entry for all cards of each set, that specifies at which pick this card is taken on average in a booster pack. (and maybe other draft related features)

Reasoning for implementation:
It might be interesting to get some more information about drafted cards. If I am not mistaken, we can currently only see the `Draft Battle evaluation` which is somewhat intransparent and "lonely" =).

**Proposed Workflow**
On the "card-page" (e.g. https://mtgarena.pro/cards/white/alseid-of-life-s-bounty-thb-1/) there is a section `evaluation`. In this section some more numbers could be added. For example the above mentioned time at which this card is picked from a pack.

**Comment by Razviar** [comment]
Maybe I'm not getting it right, but are you talking about evaluation by draft picks?
We have this: https://mtgarena.pro/cards/#rulings-1;srt-2;direct-0

**Comment by TaZi-loh** [comment]
Yeah, I know the DP metric. I interpreted it as the probability of the card get firstpicked which is nice. But not superinformative. It would be could to have a bit more information here. For example as I mentioned the average time a card is picked (or even the full distribution).
